### DISCUSSION TITLE
How to watch in a directory for new files

### DISCUSSION THREAD

**Discussion by Archie-2021** [twpayne/chezmoi#3534] [Q&A]
How can I check for new files added in certain directory. Lets say I creat a new script in my `~./local/bin` and I forget to add to chezmoi. I would not show up in `chezmoi  status` how can I make sure I have added all files in certain pattern?

**Comment by halostatue** [comment]
`chezmoi unmanaged ~/.local/bin` is your best bet. If you have a lot of things that get installed there and not just your own scripts (e.g., you use pipx or something similar), you probably want to add the files you do not want to manage to `.chemzoiignore`.

**Reply by Archie-2021** [reply]
Awesome, thanks

**Comment by twpayne** [comment]
Another possibility is to make `~/.local/bin` an [`exact_` directory](https://www.chezmoi.io/reference/source-state-attributes/). This will cause chezmoi to **remove** any files which are not managed by chezmoi, which means that `chezmoi status` will report a ` D` status for them. Note that `chezmoi apply` will then remove any unmanaged files, which is probably not what you want.

**Reply by Archie-2021** [reply]
Yeah, this one is probably a overkill for this. I wrote an script using chezmoi unmanaged that lists a bunch of interesting directories there, then I filter out files I dont care and can see the remaining. But it can be an interesting feature to add in case other people feel the same need to such a thing

Thanks
